modules/lotameIdSystem.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * This module adds LotameId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/lotameIdSystem
+ * @requires module:modules/userId
+ */
+import * as utils from '../src/utils.js';
+import { ajax } from '../src/ajax.js';
+import { submodule } from '../src/hook.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+const KEY_LAST_UPDATE = '_lota_last';
```

Comment:
could this also be stored in our pano_bundle?

Reply:
Probably, it was simpler to keep separate entries especially since the bundle part is still tbd

Reply:
I removed it altogether now we use the expiry_ts from the endpoint